examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -21,11 +21,12 @@
      "name": "stdout",
```

Comment:
<p>@OriolAbril I added bins here in favor of better visualization, because bins in auto mode, had x_ticks from 0 to 400, which made it impossible to visualize the accept 1 hist plot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>